Makefile

```diff
@@ -26,7 +26,8 @@ install_deps:
 		python3-yaml python3-coverage python3-dev pkg-config libnl-genl-3-dev \
 		libnl-route-3-dev python3-attr python3-distutils-extra python3-requests \
 		python3-requests-unixsocket python3-jsonschema python3-apport \
-		python3-bson xorriso isolinux python3-aiohttp probert cloud-init ssh-import-id
+		python3-bson xorriso isolinux python3-aiohttp probert cloud-init ssh-import-id \
+		curtin
```

Comment:
By itself this is fine, but see comment for why I think this isn't what you want.